modules/cert-manager-acme-dns-providers.adoc

```diff
@@ -16,3 +16,8 @@ The {cert-manager-operator} supports the following DNS-01 providers for ACME iss
 The {cert-manager-operator} does not support using Azure Active Directory (Azure AD) pod identities to assign a managed identity to a pod.
 ====
 * Google Cloud DNS
+* Webhook
++
+cert-manager also supports out-of-tree DNS providers using an external webhook.
```

Comment:
```suggestion
The {cert-manager-operator} also supports out-of-tree DNS providers using an external webhook.
```

Reply:
@snarayan-redhat @lunarwhite `{cert-manager-operator}` resolves to:  **cert-manager Operator for Red Hat OpenShift**
Is using that correct in this context or should we stay with **cert-manager**?

Reply:
I guess, as far as my review goes, I was thinking that I'd like to see a "the" and an "Operator" after "cert-manager". I can appreciate the fully resolved attribute may not be what we want here.